### ISSUE TITLE
Quest bugfixes version 0.6.9

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#216]
```
Cervando stated here:
http://andors.techby2guys.com/viewtopic.php?f=3&t=785

After returning the dagger to where it belongs, the conversation with the thief 
does not allow you to tell him that you have returned it, only to say you will 
take his quest, which cannot be completed, or that you won't. I feel a that the 
options should change after completion to reflect you have finished the other 
quest to return it.

Fruit Fly stated here:
http://andors.techby2guys.com/viewtopic.php?f=3&t=782

In the new quest when you have to run back and forth between prim and 
blackwater the game allows you to skip a segment. When the guy from prim sends 
you to look for battle plans in blackwater, you can simply talk to him again 
without going to blackwater, and say you found the plans. Not a big deal, but a 
bug
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 9 Jul 2011 at 7:34

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. Please try to separate things that are separate into 
separate issues.

The first issue stated above is now fixed in r122.

The second issue is fixed in r119.

Marking this issue as ready for the next version.
```

Original comment by `oskar.wi...@gmail.com` on 13 Jul 2011 at 8:24
- Changed state: **Ready**
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
